src/frontend/screens/Settings/components/Tools/index.tsx

```diff
@@ -19,12 +19,12 @@ export default function Tools() {
   const { platform } = useContext(ContextProvider)
   const isWindows = platform === 'win32'
 
-  if (isDefault || isWindows) {
+  if (isDefault || isWindows || !runner) {
     return <></>
   }
 
   type Tool = 'winecfg' | string
-  async function callTools(tool: Tool, exe?: string) {
+  const callTools = async (tool: Tool, exe?: string) => {
```

Comment:
I had to change these to arrow functions because the typescript complains about `runner` being possible undefined inside `function()` (even though this happens after the `if`)

looks like, because `this` is different in the 2 cases, runner can be undefined for named functions but it's correct with arrow functions